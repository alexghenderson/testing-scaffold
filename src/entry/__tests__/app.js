import React from 'react';
import {render} from 'react-testing-library';

import App from '/entry/app';

describe('App Component', () => {
  test('renders', () => {
    const {container} = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
