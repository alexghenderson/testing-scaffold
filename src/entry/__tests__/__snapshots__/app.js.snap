// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders 1`] = `
<div>
  <div>
    Hello world
  </div>
</div>
`;
